app/src/main/java/org/mifos/mobile/api/services/AuthenticationService.kt

```diff
@@ -0,0 +1,17 @@
+package org.mifos.mobile.api.services
+
+import io.reactivex.Observable
+import org.mifos.mobile.api.ApiEndPoints
+import org.mifos.mobile.models.User
+import org.mifos.mobile.models.payload.LoginPayload
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+/**
+ * @author Vishwajeet
+ * @since 09/06/16
+ */
+interface AuthenticationService {
+    @POST(ApiEndPoints.AUTHENTICATION)
```

Comment:
Add an empty line here, to enhance the readability of code. Do the same changes in other places also, if this mistake is done.

Reply:
Sorry, this seems to have been repetitive. Corrected all such occurrences.